Week1/homework/main.js

```diff
@@ -0,0 +1,263 @@
+    mahkemelerde: 'https://motherinlondon.files.wordpress.com/2017/09/img_4782.jpg?w=293&h=391',
+    masumiyet_müzesi: 'https://upload.wikimedia.org/wikipedia/en/b/b1/Masmiyetmuzesi.jpg',
+    bir_kedi_bir_adam_bir_ölüm: 'https://i.dr.com.tr/cache/500x400-0/originals/0000000390208-1.jpg',
+    son_ada: 'https://i.dr.com.tr/cache/600x600-0/originals/0000000457487-1.jpg',
+    serenad:
+      'https://images-na.ssl-images-amazon.com/images/I/51b1emDMkrL._SX344_BO1,204,203,200_.jpg',
+    the_da_vinci_code: 'https://upload.wikimedia.org/wikipedia/en/6/6b/DaVinciCode.jpg',
+    endgame:
+      'https://d243y1uga1q3sn.cloudfront.net/assets/files/302874/endgame-paperback-cover-9781782112617.jpg',
+    crime_and_punishment: 'https://images.penguinrandomhouse.com/cover/9780553211757',
+  };
+
+  // --------------------------1.8---------------------------
+  // Loop over these entries (hint: Object.keys(objectName) gives you an array containing the keys). 
+  // Then write a function which places an image at the corresponding li element.
+  // Remember that objects are not ordered, so you cannot guarantee that the first key is the first li element. 
+  // (Hint: you could give each li item an id tag by modifying the function you made before.)
+
+
+  const createDiv = document.createElement("div");
+  createDiv.setAttribute("id", "newDiv");
+  const getBody = document.querySelector("body").appendChild(createDiv);
+  const ul = document.createElement("ul");
+  createDiv.appendChild(ul);
+  const keys = Object.keys(infoBooks);
+  const keysOfCovers = Object.keys(bookCovers);
+
+  const createElement =()=>{
+    for (let i = 0; i<bookTitles.length; i++){
+      const li = document.createElement("li");
+      li.setAttribute("id", keys[i])
+      ul.appendChild(li);
+      const h1 = document.createElement("h1");
+      const removeUnderscores = keys[i].replace(/_/g, " ");
+      h1.innerText = removeUnderscores;
+      li.appendChild(h1);
+      h1.style.color="#000"
+      h1.style.textTransform="uppercase";
+
+      const h2 = document.createElement("h2");
+      h2.innerText =infoBooks[keys[i]].author;
+      li.appendChild(h2);
+
+      const h3 = document.createElement("h3");
+      h3.innerText =infoBooks[keys[i]].year;
+      li.appendChild(h3);
+
+      const h4 = document.createElement("h4");
+      h4.innerText =infoBooks[keys[i]].language;
+      li.appendChild(h4);
+      h4.style.fontStyle="italic"
+
+      const createImg = document.createElement("img");
+      createImg.setAttribute("src", bookCovers[keysOfCovers[i]]);
+      createImg.setAttribute("alt", keysOfCovers[i]);
+      li.appendChild(createImg);
+
+    }
+
+  }
+  createElement()
+
+  //additional things
+
+  const newButton = document.createElement('button');
+  const rootOfMain = document.querySelector('.wrapper').appendChild(newButton);
+  newButton.setAttribute('id', 'see-more');
+  newButton.innerHTML = 'Click to See Below!';
+  newButton.style.margin = '2%';
```

Comment:
you can set classes from you JS and set your styling in css files, separation makes it cleaner